### PR TITLE
executor: fixed comments warning. 

### PR DESCRIPTION
**Pull request by zhexuany**

### CODE REVIEW

File: executor/write.go [hunk 1]
```diff
@@ -449,6 +449,7 @@ func escapeCols(strs [][]byte) []string {
 	return ret
 }
 
+// escape ...
```
Comment (shenli) [inline executor/write.go]: escape handles escape characters when running load data statement.

File: executor/write.go [hunk 2]
```diff
@@ -682,6 +683,7 @@ func (e *InsertExec) Close() error {
 	return nil
 }
 
+// getColumns ...
```
Comment (shenli) [inline executor/write.go]: getColumns gets the explicitly specified columns of an insert statement.  There are three cases:

File: executor/builder.go [hunk 3]
```diff
@@ -365,6 +365,7 @@ func (b *executorBuilder) buildUnionScanExec(v *plan.PhysicalUnionScan) Executor
 	return us
 }
 
+// buildMergeJoin ...
```
Comment (shenli) [inline executor/builder.go]: buildMergeJoin builds SortMergeJoin executor.

File: executor/show.go [hunk 4]
```diff
@@ -329,6 +329,7 @@ func (e *ShowExec) fetchShowIndex() error {
 	return nil
 }
 
+// fetchShowCharset ...
```
Comment (shenli) [inline executor/show.go]: fetchShowCharset gets all charset information and fill them into e.rows.

**Comment by zhexuany** [comment]
@shenli DONE. PTAL.

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@coocood @hanfei1991 @tiancaiamao PTAL

File: executor/adapter.go [hunk 5]
```diff
@@ -77,7 +77,7 @@ func (a *recordSet) Close() error {
 
 // statement implements the ast.Statement interface, it builds a plan.Plan to an ast.Statement.
 type statement struct {
-	// The InfoSchema cannot change during execution, so we hold a reference to it.
+	// is is the InfoSchema cannot change during execution, so we hold a reference to it.
```
Comment (coocood) [inline executor/adapter.go]:
```
// is is the InfoSchema reference. 
// The InfoSchema cannot change during execution, so we hold a reference to it.

File: executor/merge_join.go [hunk 6]
```diff
@@ -30,7 +30,7 @@ import (
 // 2. For other cases its preferred not to use SMJ and operator
 // will throw error.
 type MergeJoinExec struct {
-	// Left is always the driver side
```
Comment (coocood) [inline executor/merge_join.go]: This comment describes several fields below.

File: executor/merge_join.go [hunk 7]
```diff
@@ -43,7 +43,7 @@ type MergeJoinExec struct {
 	cursor        int
 	defaultValues []types.Datum
 
-	// Default for both side in case full join
```
Comment (coocood) [inline executor/merge_join.go]: Ditto

File: executor/metrics.go [hunk 8]
```diff
@@ -283,7 +283,7 @@ const (
 	attrAggregate = "Agg"
 )
 
-// Not all attributes is used to create the label, because we don't want too many labels.
```
Comment (coocood) [inline executor/metrics.go]: // toLabal ...

File: executor/union_scan.go [hunk 9]
```diff
@@ -66,7 +66,7 @@ func (udb *dirtyDB) getDirtyTable(tid int64) *dirtyTable {
 }
 
 type dirtyTable struct {
-	// key is handle.
+	// addedRows is a map whose key is handle.
```
Comment (coocood) [inline executor/union_scan.go]: // addedRows ...
// The key is handle.

File: executor/write.go [hunk 10]
```diff
@@ -1091,7 +1093,7 @@ type UpdateExec struct {
 	SelectExec  Executor
 	OrderedList []*expression.Assignment
 
-	// Map for unique (Table, handle) pair.
+	// updatedRowKeys is map for unique (Table, handle) pair.
```
Comment (coocood) [inline executor/write.go]: is a map

File: executor/distsql.go [hunk 11]
```diff
@@ -68,7 +68,8 @@ type lookupTableTask struct {
 
 	// The handles fetched from index is originally ordered by index, but we need handles to be ordered by itself
 	// to do table request.
-	// The indexOrder map is used to save the original index order for the handles.
+
```
Comment (coocood) [inline executor/distsql.go]: The above comment is used to comment `indexOrder`, we should not add an empty line.

File: executor/executor.go [hunk 12]
```diff
@@ -81,6 +81,7 @@ const (
 	codeErrBuildExec    terror.ErrCode = 9
 	codeBatchInsertFail terror.ErrCode = 10
 	// MySQL error code
+
```
Comment (coocood) [inline executor/executor.go]: Here is the downside of `goword`, when we need to comment several fields, goword makes it harder to read.

**Comment by coocood** [comment]
LGTM

**Pull request merged**